atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -288,8 +285,32 @@ public void compactInternally() {
                 throw new RuntimeException(e);
             } finally {
                 log.info("Call to KVS.compactInternally on table {} took {} ms.",
-                        tableRef, timer.elapsed(TimeUnit.MILLISECONDS));
+                        LoggingArgs.tableRef(tableRef), timer.elapsed(TimeUnit.MILLISECONDS));
             }
         }
     }
+
+    private void runShrinkSpaceCompact() throws TableMappingNotFoundException {
+        Stopwatch shrinkAndCompactTimer = Stopwatch.createStarted();
+        conns.get().executeUnregisteredQuery(
+                "ALTER TABLE " + oracleTableNameGetter.getInternalShortTableName(conns, tableRef)
+                        + " SHRINK SPACE COMPACT");
+        log.info("Call to SHRINK SPACE COMPACT on table {} took {} ms.",
+                tableRef, shrinkAndCompactTimer.elapsed(TimeUnit.MILLISECONDS));
```

Comment:
Realise this was extracted from the original, but you probably want `LoggingArgs` on the table-ref, and `SafeArg` on the stopwatch.